css/main.css

```diff
@@ -0,0 +1,595 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+a {
+  text-decoration: none;
+}
+
+hr {
+  width: 80vw;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 15px;
+  background-color: #3c3a39;
+  color: white;
+}
+
+#desktop-nav,
+#desktop-nav ul {
+  display: none;
+}
+
+#logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.menu-bars {
+  background: transparent;
+  border: none;
+  outline: none;
+}
+
+.menu-bars i {
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+#mobile-nav {
+  z-index: 9999;
+  position: absolute;
+  background-color: black;
+  width: 100%;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  margin: 0;
+  padding: 0;
+  display: none;
+  padding-top: 50px;
+}
+
+#close-button-container {
+  width: 100%;
+  display: flex;
+  justify-content: flex-end;
+}
+
+#nav-close {
+  text-align: right;
+}
+
+#mobile-nav a {
+  font-size: 32px;
+  color: rgb(207, 194, 194);
+  display: flex;
+  justify-content: flex-start;
+}
+
```

Comment:
Kindly reconsider your design as there still exists a big difference in the style as compared to the Figma Design.

For better understanding I am sharing the link to a specific Figma design:
[Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=dttQe0Ya1jzMx73q-0)